packages/babel-plugin/src/__test__/k.test.ts

```diff
@@ -74,6 +74,34 @@ describe("k api", () => {
         expect(getExpectSnapshot(result)).toMatchSnapshot();
       });
 
+      test("using pseudo elements with conditional should match snapshot", () => {
+        // Arrange
+        const inputCode = `
+        import { k } from '@kuma-ui/core'
+        function App({ flag }) {
+          return <k.div p={2} m={flag ? 100 : 10} _hover={{ color: flag ? 'blue' : 'red', backgroundColor: flag ? 'gray' : 'white' }} />
+        }
+      `;
+        // Act
+        const result = babelTransform(inputCode);
+        // Assert
+        expect(getExpectSnapshot(result)).toMatchSnapshot();
+      });
+
+      test("using pseudo elements with top-level conditional should match snapshot", () => {
+        // Arrange
+        const inputCode = `
+        import { k } from '@kuma-ui/core'
+        function App({ flag }) {
+          return <k.div p={2} m={flag ? 100 : 10} _hover={flag ? { color: 'blue', bgColor: 'gray'} : {color: 'green'}} />
```

Comment:
Note: See the snapshot for how this is compiled. Note that conditional expression can appear at the top level of pseudo props or per prop level. If conditional expressions are nested, it will bail out